crates/sovereign-sdk/rollup-interface/src/node/rpc/mod.rs

```diff
@@ -383,9 +380,14 @@ where
         {
             let mut btree_map = BTreeMap::new();
             while let Some((key, value)) = map.next_entry::<String, Option<String>>()? {
-                let key = hex::decode(&key).map_err(A::Error::custom)?;
+                let key = key.trim_start_matches("0x");
+                let key = hex::decode(key).map_err(A::Error::custom)?;
+
                 let value = match value {
-                    Some(value) => Some(hex::decode(&value).map_err(A::Error::custom)?),
+                    Some(value) => {
+                        let value = value.trim_start_matches("0x");
+                        Some(hex::decode(value).map_err(A::Error::custom)?)
```

Comment:
https://docs.rs/faster-hex/latest/faster_hex/fn.deserialize.html : Serde: Deserialize with 0x-prefix and ignore case

> Compare with [hex](https://crates.io/crates/hex):
    Encoding ~10x over
    Decoding ~10x over